metabase/metabase
# fix goal line on scatter plot

Closes https://github.com/metabase/metabase/issues/36584

### Description

Adding a goal line to the scatter plot caused an extra line series to appear. This PR fixes that by making that series invisible.

The fix is in `goal-line.ts`, the rest the PR is just adding a storybook example.

### How to verify

1. Create any scatter plot
2. Enable goal line in viz settings
3. Add to dashboard and send a subscription
4. Confirm nothing extra besides the scatter points and goal line appear on the chart

### Demo

![Screenshot 2023-12-14 at 2.28.20 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/e7da0b27-46bc-4a16-901c-0e8bb29c1fb5.png)

### Checklist
